### PR TITLE
link template works on mac, ios, android studio 

### PR DESCRIPTION
**Pull request by drelaptop**
just like those default template do, only with little difference.

default template on mac, #1300 
default template on ios, #1304 
default template on android studio, #1307

### CODE REVIEW

**Comment by drelaptop** [comment]
please review and merge @minggo

File: templates/js-template-link/frameworks/runtime-src/Classes/AppDelegate.cpp [hunk 1]
```diff
@@ -44,7 +43,7 @@
 
 USING_NS_CC;
 
-AppDelegate::AppDelegate() : Application("js-test")
+AppDelegate::AppDelegate(int width, int height) : Application("Cocos Runtime", width, height)
```
Comment (minggo) [inline templates/js-template-link/frameworks/runtime-src/Classes/AppDelegate.cpp]: `Cocos Runtime` is not suitable here.

Comment (drelaptop) [reply templates/js-template-link/frameworks/runtime-src/Classes/AppDelegate.cpp]: I think so, hard coded the application name is a bad choice, might we can add a macro
```
// modify it to change application name
#define COCOS_APPLICATION_NAME "Cocos Game"
```
for further, we can modify it by cocos console, for example change default value to `HelloJavaScript`, change it to `hellowrold` when the identifier is `org.cocos2d.helloworld`

**Comment by drelaptop** [comment]
changed "Cocos Runtime" into "Cocos Game", I think "Cocos Game" is a better app name than "js-tests". please review @minggo

**Comment by minggo** [comment]
Yep, `Cocos Game` is good.

**Pull request merged**